eap-xp4-openjdk11-image-stream.json

```diff
@@ -0,0 +1,123 @@
+{
+    "kind": "List",
+    "apiVersion": "v1",
```

Comment:
actually, there is a change I forgot to ask you.

Could you please a namespaced apiVersion to avoid a warning

```
"apiVersion": "image.openshift.io/v1"
```

(similar to https://issues.redhat.com/browse/JBEAP-23082)

Reply:
Oh, does the List get this as well, when I looked at https://issues.redhat.com/browse/JBEAP-23082 that wasn't changed, I'll double check.

Reply:
d'oh, this `apiVersion` is for the list. I did not pay attention, sorry

Reply:
sorry for the noise, "v1" for the "List" kind is correct